.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,53 @@
+name: "Core features"
+description: "Suggest an idea for the project"
+title: "[Core feature] "
+labels: [enhancement,untriaged]
+body:
+  - type: textarea
+    id: motivation
+    validations:
+      required: true
+    attributes:
+      label: " Motivation: Why do you think this is important?"
+      description: " A clear and concise description of what the problem is."
+      placeholder: " Ex. I'm always frustrated when [...] OR why this would be a cool addition OR which usecases could this unblock."
```

Comment:
```suggestion
      placeholder: "I'm frustrated when [...] OR this feature can be a cool addition because [...] OR this feature can simplify [...] use case."
```